src/components/MessageDisplay.svelte

```diff
@@ -17,6 +17,7 @@
     mchadUsers,
     showTimestamp,
     spotlightedTranslator,
+    sessionHidden,
     ytcDeleteBehaviour
```

Comment:
There's a bunch of unused imports here now that they're refactored out. This is just one of them, GitHub ain't letting me highlight "unchanged" code.

Reply:
Fixed in dd0d826